### PR TITLE
[UiBundle] Fix engine interface deprecation

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | partially #10928 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/UiBundle/Controller/SecurityController.php [hunk 1]
```diff
@@ -71,11 +71,16 @@ public function loginAction(Request $request): Response
...
+        $content = $this->templatingEngine->render($template, [
             'form' => $form->createView(),
             'last_username' => $lastUsername,
             'last_error' => $lastError,
         ]);
+
+        $response = new Response();
+        $response->setContent($content);
+
+        return $response;
```
Comment (Zales0123) [inline src/Sylius/Bundle/UiBundle/Controller/SecurityController.php:83]:
```php
return new Response($content);
```

**Review by Zales0123 — COMMENTED** [review]
👍

File: src/Sylius/Bundle/UiBundle/Controller/SecurityController.php (hunk 1 above)
Comment (loic425) [reply src/Sylius/Bundle/UiBundle/Controller/SecurityController.php:83]: good idea

Comment (loic425) [reply src/Sylius/Bundle/UiBundle/Controller/SecurityController.php]: @Zales0123 Fixed ;)

**Review by Zales0123 — COMMENTED** [review]
Oh, in fact, it should be done like [this](https://github.com/Sylius/Sylius/pull/12084/files#diff-c2d70c1009057d75487e2317084a44fd8bd9ba222d71ee4f5cef693ff2ba91a2R35) to keep the BC :/ So, as the mentioned PR will be merged in the very nearest future, I would close this one but still, thank you very much for your contribution 🎉

**Comment by loic425** [comment]
> Oh, in fact, it should be done like [this](https://github.com/Sylius/Sylius/pull/12084/files#diff-c2d70c1009057d75487e2317084a44fd8bd9ba222d71ee4f5cef693ff2ba91a2R35) to keep the BC :/ So, as the mentioned PR will be merged in the very nearest future, I would close this one but still, thank you very much for your contribution 🎉

Yes, I didn't see these changes on this PR. Thx.

**Pull request closed without merging**